native_client/ctcdecode/third_party/openfst-1.6.7-win/openfst-1.6.7/.editorconfig

```diff
@@ -0,0 +1,11 @@
+; EditorConfig to support per-solution formatting.
```

Comment:
Is this editor config file really needed ?

Reply:
And generally speaking, we need a `README.mozilla` that documents how to import / update this.

Reply:
Even if there's not change ; it's important we know how / where you did pick up things

Reply:
See for example [native_client/kenlm/README.mozilla](https://github.com/mozilla/DeepSpeech/blob/master/native_client/kenlm/README.mozilla)